crypto/ledger/ledger_secp256k1.go

```diff
@@ -210,11 +212,16 @@ func warnIfErrors(f func() error) {
 }
 
 func convertDERtoBER(signatureDER []byte) ([]byte, error) {
-	sigDER, err := btcec.ParseDERSignature(signatureDER, btcec.S256())
+	sigDER, err := ecdsa.ParseDERSignature(signatureDER)
 	if err != nil {
 		return nil, err
 	}
-	sigBER := tmbtcec.Signature{R: sigDER.R, S: sigDER.S}
+
+	sigStr := sigDER.Serialize()
+	var r, s big.Int
+	r.SetBytes(sigStr[4 : 4+sigStr[3]])
+	s.SetBytes(sigStr[4+sigStr[3]+2:])
```

Comment:
Perhaps a comment is warranted? A test would be nice as well. I'm surprised no existing test caught this.

Reply:
Adding a comment. This is tested, however, you simply need to call the tests with the `ledger` and `ledger_mock` flag.